packages/react/src/components/iconBadge/IconBadge.tsx

```diff
@@ -51,16 +45,12 @@ const TypeColorMapping: Record<IconBadgeType, string> = {
     [IconBadgeType.Major]: 'mod-major',
 };
 
-export const IconBadge: FunctionComponent<IconBadgeProps> = ({
-    svgName,
-    type,
-    size = IconBadgeSize.Medium,
-    svgClass,
-    className,
-}) => (
-    <Svg
-        className={classNames('icon-badge', SizeClassMapping[size], TypeColorMapping[type], className)}
-        svgName={svgName}
-        svgClass={classNames('icon align-middle', SizeClassMapping[size], svgClass)}
-    />
-);
+export const IconBadge: FunctionComponent<IconBadgeProps> = ({icon, type, size = IconBadgeSize.Medium, className}) => {
+    const IconName = icon;
+
+    return (
+        <div className={classNames('icon-badge', SizeClassMapping[size], TypeColorMapping[type], className)}>
+            <IconName height={24} className={'align-middle'} />
```

Comment:
The examples use the 16 px size variant of the icons but here we set 24px

Reply:
I can use the 24px variant if you prefer 🤷‍♀️

Reply:
I think that would be more appropriate yes, otherwise the strokes look more thick then they should be compared to the other icons

Reply:
You got it then 😄